src/test/resources/alma-fix/990150856900206441.json

```diff
@@ -0,0 +1,128 @@
+    "label" : "Webseite der hbz-Ressource 990150856900206441",
+    "type" : [ "BibliographicDescription" ],
+    "dateCreated" : "20210406",
+    "dateModified" : "20220723",
+    "inDataset" : {
+      "id" : "http://lobid.org/resources/dataset#!",
+      "label" : "lobid-resources – Der hbz-Verbundkatalog als Linked Open Data"
+    },
+    "resultOf" : {
+      "type" : [ "CreateAction" ],
+      "endTime" : "0000-00-00T00:00:00",
+      "instrument" : {
+        "id" : "https://github.com/hbz/lobid-resources",
+        "type" : [ "SoftwareApplication" ],
+        "label" : "Software lobid-resources"
+      },
+      "object" : {
+        "id" : "https://lobid.org/hbz01/990150856900206441",
+        "type" : [ "DataFeedItem" ],
+        "label" : "hbz-Ressource 990150856900206441 im Exportformat MARC21 XML",
+        "inDataset" : {
+          "id" : "https://datahub.io/dataset/hbz_unioncatalog",
+          "label" : "hbz_unioncatalog"
+        }
+      }
+    },
+    "license" : [ {
+      "id" : "http://creativecommons.org/publicdomain/zero/1.0",
+      "label" : "Creative Commons-Lizenz CC0 1.0 Universal"
+    } ],
+    "sourceOrganization" : {
+      "id" : "http://lobid.org/organisations/DE-Kn28#!",
+      "label" : "lobid Organisation"
+    },
+    "provider" : {
+      "id" : "http://lobid.org/organisations/DE-Kn28#!",
+      "label" : "lobid Organisation"
+    }
+  },
+  "sameAs" : [ {
+    "id" : "http://hub.culturegraph.org/resource/HBZ-TT002001274",
+    "label" : "Culturegraph Ressource"
+  }, {
+    "id" : "http://worldcat.org/oclc/831219119",
+    "label" : "OCLC Ressource"
+  } ],
+  "isPartOf" : [ {
+    "type" : [ "IsPartOfRelation" ],
+    "hasSuperordinate" : [ {
+      "id" : "http://lobid.org/resources/HT001375579#!",
+      "label" : "lobid Ressource"
+    } ],
+    "numbering" : "1"
+  } ],
+  "language" : [ {
+    "id" : "http://id.loc.gov/vocabulary/iso639-2/ger",
+    "label" : "Deutsch"
+  }, {
+    "id" : "http://id.loc.gov/vocabulary/iso639-2/lat",
+    "label" : "Latein"
+  }, {
+    "id" : "http://id.loc.gov/vocabulary/iso639-2/gre",
+    "label" : "Neugriechisch"
+  } ],
+  "extent" : "XVI, 323 S.",
+  "hasItem" : [ {
+    "id" : "https://lobid.org/item/990150856900206441",
+    "type" : [ "Item", "MBD" ],
+    "heldBy" : {
+      "id" : "http://lobid.org/organisations/DE-605#!",
+      "label" : "lobid Organisation"
+    },
+    "label" : "990150856900206441"
+  } ],
+  "medium" : [ {
+    "label" : "Print",
+    "id" : "http://rdaregistry.info/termList/RDAproductionMethod/1010"
+  } ],
+  "type" : [ "BibliographicResource", "EditedVolume", "Book" ],
+  "responsibilityStatement" : [ "von Hans Lietzmann" ],
+  "contribution" : [ {
+    "agent" : {
+      "gndIdentifier" : "(DE-588)118572865",
+      "id" : "https://d-nb.info/gnd/118572865",
+      "label" : "Lietzmann, Hans",
+      "type" : [ "Person" ],
+      "dateOfBirth" : "1875",
+      "dateOfDeath" : "1942"
```

Comment:
already covered?

Reply:
The edge case is the second contribution object which adds zeros at the beginning of years that have less than 4 digits.